src/include/simeng/arch/aarch64/helpers/neon.hh

```diff
@@ -187,10 +187,10 @@ RegisterValue vecExtVecs_index(
   const uint64_t index = static_cast<uint64_t>(metadata.operands[3].imm);
   T out[16 / sizeof(T)] = {0};
 
-  for (int i = index; i < I; i++) {
+  for (uint64_t i = index; i < I; i++) {
```

Comment:
This will have been highlighted due to a warning. The issue is the mixing of signed and unsigned types during comparisons, not the size of the variable. As discussed offline this morning we should be changing instance like 148 but not as part of this PR specifically